ext/intl/uchar/uchar.c

```diff
@@ -251,7 +251,7 @@ IC_METHOD(charName) {
 
 	if ((zend_parse_parameters(ZEND_NUM_ARGS(), "z|l", &zcp, &nameChoice) == FAILURE) ||
 	    (convert_cp(&cp, zcp) == FAILURE)) {
```

Comment:
Please split off the convert_cp() check into a separate branch.

Reply:
I wasn't exactly sure how to go with these cases, but your suggestion sounds good! And this way I'll be able to also include similar cases where the return value is currently false.

Reply:
I've just pushed a fix and I hope that I got it right this time.